src/main/java/com/epam/ta/reportportal/core/user/UserPasswordService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.ta.reportportal.core.user;
+
+import com.epam.ta.reportportal.entity.user.User;
+import com.epam.ta.reportportal.ws.model.user.ChangePasswordRQ;
+import com.epam.ta.reportportal.ws.model.user.CreateUserRQFull;
+import com.epam.ta.reportportal.ws.model.user.ResetPasswordRQ;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public interface UserPasswordService {
```

Comment:
Can't see need of this interfacel. The idea of interfaces that it can be used in several places. But you created methods that does the same (password encoding) but gets different parameters. If you want to create an interface for it, so it's better to create one with the method 'encode', that receives string as parameter and returns encoded string back.

Reply:
fixed